azure/functions/decorators/function_app.py

```diff
@@ -2740,6 +2790,379 @@ def decorator():
+        output object in function code.
+        :param connection_name: app setting or environment variable which
+        contains a connection string value.
+        :param collection: The name of the collection or table to search or
+        store.
+        :param query: The semantic query text to use for searching.
+        :param embeddings_model: The ID of the model to use for embeddings.
+        The default value is "text-embedding-ada-002".
+        :param chat_model: The name of the Large Language Model to invoke for
+        chat responses. The default value is "gpt-3.5-turbo".
+        :param system_prompt: Optional The system prompt to use for prompting
+        the large language model.
+        :param max_knowledge_count: Optional. The number of knowledge items to
+        inject into the SystemPrompt. Default value: 1
+        :param data_type: Optional. Defines how Functions runtime should treat
+        the parameter value. Default value: None
+        :param kwargs: Keyword arguments for specifying additional binding
+        fields to include in the binding json
+
+        :return: Decorator function.
+        """
+
+        @self._configure_function_builder
+        def wrap(fb):
+            def decorator():
+                fb.add_binding(
+                    binding=SemanticSearchInput(
+                        name=arg_name,
+                        connection_name=connection_name,
+                        collection=collection,
+                        query=query,
+                        embeddings_model=embeddings_model,
+                        chat_model=chat_model,
+                        system_prompt=system_prompt,
+                        max_knowledge_count=max_knowledge_count,
+                        data_type=parse_singular_param_to_enum(data_type,
+                                                               DataType),
+                        **kwargs))
+                return fb
+
+            return decorator()
+
+        return wrap
+
+    def embeddings_store_output(self,
+                                arg_name: str,
+                                input: str,
+                                input_type: InputType,
+                                connection_name: str,
+                                collection: str,
+                                model: Optional[OpenAIModels] = OpenAIModels.DefaultEmbeddingsModel,  # NoQA
+                                max_chunk_length: Optional[int] = 8 * 1024,
+                                max_overlap: Optional[int] = 128,
+                                data_type: Optional[
+                                    Union[DataType, str]] = None,
+                                **kwargs) \
+            -> Callable[..., Any]:
+        """
+        Supported list of embeddings store is extensible, and more can be
+        added by authoring a specially crafted NuGet package. Visit the
+        currently supported vector specific folder for specific usage
```

Comment:
any link to be added here?

Reply:
I intentionally didn't add the link for individual stores. Didn't seem very useful